latch_cli/services/cp/throttle.py

```diff
@@ -0,0 +1,15 @@
+from multiprocessing.managers import BaseManager
+from typing import Type
+
+from attr import dataclass
+
+
+@dataclass
+class Throttle:
+    delay: float = 0
+
+    def get_delay(self):
+        return self.delay
+
+    def set_delay(self, d: float):
+        self.delay = d
```

Comment:
needed for `multiprocessing` manager so that more than one process can interact with it

also i believe theres a built in Value in `multiprocessing.SyncManager` but i think this is more readable